data/models/2x-AnimeJaNai_V2.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "AnimeJaNai V2",
```

Comment:
This isn't really how the model database works. The idea is that each model represents a single checkpoint (and any of its conversions to different platforms like NCNN and ONNX). 

So you have to make 3 models for AnimeJaNai V2. Compact, UltraCompact, and SuperUltraCompact should all get their own model. You can all link them to the same .zip file though.

The reason we do this is searchability. Some people might search for super fast/lite models and configure the search to ignore the rest. Other people might search for heavier models and ignore lite ones because they tend to give worse results.

Reply:
Sorry that we didn't mention this anywhere. I added it. #267.